### DISCUSSION TITLE
Do not save or delete a file immediately

### DISCUSSION THREAD

**Discussion by beginner000001** [obsproject/obs-studio#6336] [Development Questions] [answered]
Does it only happen to me that I screw up a recording and don't want to finish saving it right away. If this is the case for others, then a shortcut that either does not save the recording or removes it immediately after saving would be very helpful. In reality I always have to go into the folder and remove the file manually and with my lack of talent I have to delete a large amount of files each time....
Regardless of whether this is implemented or not, I would like to thank the team from the bottom of my heart for this excellent software! Really thank you thank you thank you.

**Comment by Fenrirthviti** [comment, marked as answer]
Adding a programmatic way to delete a file is not something we will be adding to OBS. File deletion should be an explicit and deliberate act, not something you can accidentally hit with either a hotkey or button or menu in OBS. We understand that some people feel it would be more convenient and they personally wouldn't accidentally delete anything, but it is simply too dangerous of a feature for us to include.

**Reply by beginner000001** [reply]
ok, I understand. 
It also would help to add a suffix to the filname e.g. "_crap" or "_rubbish". 
For me it would be much easier, I would not have to open about 50 files individually and check whether they are obsolete. 
Anyhow thanks for thinking about this.

**Reply by sukrosono** [reply]
[did you know](https://obsproject.com/forum/threads/does-obs-have-save-file-name-as-feature-popping-up-after-clicking-stop-record.137675/#:~:text=OBS%20is%20just%20finalizing%20and,use%20to%20customize%20the%20name.) ?
Start your recording with good vibes on the filename, instead of rubbish, etc.
